config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get    "/auth/:provider/callback", to: "sessions#create"
+  get    "/auth/failure",            to: "sessions#failure"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.